media-libs/glm/glm-0.9.7.1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="OpenGL Mathematics"
+HOMEPAGE="http://glm.g-truc.net/"
+SRC_URI="https://github.com/g-truc/glm/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
```

Comment:
```
GLM can be distributed and/or modified under the terms of either
a) The Happy Bunny License, or b) the MIT License.
```
https://github.com/g-truc/glm/blob/master/copying.txt#L4

we should leave the choice to the user